sippy-ng/src/tests/TestTable.js

```diff
@@ -322,6 +342,37 @@ function TestTable(props) {
         </div>
       ),
     },
+    delta_from_average: {
+      field: 'delta_from_average',
+      headerName: 'Delta_From_Average',
+      type: 'number',
+      renderCell: (params) => {
+        return (
+          <div className="percentage-cell">
+            {params.value
+              ? Number(params.value).toFixed(1).toLocaleString() + '%'
+              : ''}
+          </div>
+        )
+      },
+    },
+    working_average: {
+      field: 'working_average',
+      headerName: 'Working average for test on all NURPs',
```

Comment:
Lets hide this column, it looks like it's always just the inverse of the delta from average.

Reply:
This is the average from which delta is calculated. It is only the inverse when rate is 0. I think it makes sense to keep.